Composer/packages/types/src/auth.ts

```diff
@@ -30,3 +30,12 @@ export type WebAuthParameters = {
   /** List of OAuth scopes that will be granted once the user has authenticated. */
   scopes?: string[];
 };
+
+export type CurrentUser = {
+  token: string; // aad token
+  graph: string | null; // graph token
+  email?: string;
+  name?: string;
+  expiration?: number;
```

Comment:
Can we expose this in the auth API that plugins use?

Reply:
this is available from `useAuthApi()` in the plugin context.